apps/src/lib/levelbuilder/code-docs-editor/ProgrammingExpressionsTable.jsx

```diff
@@ -171,6 +186,19 @@ export default function ProgrammingExpressionsTable({
   const renderFilters = () => {
     return (
       <>
+        <select
+          onChange={e => {
+            setSelectedResultType(e.target.value);
+            setCurrentPage(1);
+          }}
+          value={selectedResultType}
```

Comment:
nit: mind adding a margin to the right so that the dropdowns have a bit of space between them (like the dropdown below)

Reply:
good catch- done!